src/main/java/com/labsynch/labseer/api/ApiCmpdSearchController.java

```diff
@@ -79,7 +81,8 @@
 		headers.setExpires(0); // Expire the cache
 		
 		try {
-			String searchResults = searchFormService.findParentIds(molStructure, maxResults, similarity, searchType, outputFormat);
+			SearchType matchedSearchType = SearchType.getIfPresent(searchType).orElse(SearchType.SUBSTRUCTURE);
```

Comment:
Because on line 111 of the previous version. It had explicitly set SUBSTRUCTURE if null or ""

Old version:
```
if (searchType == null || searchType.equalsIgnoreCase("")) searchType = "SUBSTRUCTURE";
```

I don't have a strong plan around DEFAULT, its just that didn't like passing around null. I can't imagine why we ever wanted to leave it up to the service to decide.  Mostly this results in an error but a few of the implmentations do pick a default.